conf/config-default.yaml

```diff
@@ -246,6 +246,7 @@ plugins:                          # plugin list (sorted in alphabetical order)
   - wolf-rbac
   - zipkin
   # - server-info
+  - traffic-split
```

Comment:
Better to disable this plugin by default.

Reply:
This plugin does not involve changes to the core code, I think we can enable.